manifest.yaml

```diff
@@ -1,7 +1,7 @@
 id: nextcloud 
 title: Nextcloud
-version: 24.0.4.3
-release-notes: Initial version for EmbassyOS
+version: 24.0.4.5
+release-notes: Updated to run on x86_64 architecture
```

Comment:
Would note just that this is the "Initial release for embassyOS"

Reply:
changed